### ISSUE TITLE
Back gesture on replies doesn't return to comments

### ISSUE THREAD

**Issue by 0xrxL** [libre-tube/LibreTube#6010]
### Describe your suggested feature

When I open comments list and also open the answers below a comment, I found more comfortable (with device one hand use) to make a gesture to return to the previous page, instead of tapping on the dedicated button.

Unfortunately I can't do that with LibreTube, because a go back gesture close the entire comment page, independently if I'm on the comments list or answers list.

It would be useful to add the possibility to return to all comments page (from the answers one) with the system gesture, and eventually close the entire comments page once the view is focused on the main comments page.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by DahilerMehvOlur** [comment]
Yes It is uncomfortable thing. Because I'm used to using the system's way of going back to go back.
